src/utils/MainApi.js

```diff
@@ -0,0 +1,78 @@
+export const BASE_URL = "http://localhost:3000";
+// export const BASE_URL =  "api.angelique.students.nomoredomainssbs.ru";
+
+function checkResponse(res) {
+  return res.ok ? res.json() : Promise.reject(`Error: ${res.status}`);
+};
+
```

Comment:
* When I search something there is a `key` error https://disk.yandex.ru/i/xPVg2VP3T9XiMw